internal/output/spdx.go

```diff
@@ -0,0 +1,28 @@
+package output
+
+import (
+	"encoding/json"
+	"io"
+
+	scalibr "github.com/google/osv-scalibr"
+	"github.com/google/osv-scalibr/converter"
+	"github.com/google/osv-scanner/v2/pkg/models"
+)
+
+// PrintSPDXResults writes results to the provided writer in SPDX format
+func PrintSPDXResults(vulnResult *models.VulnerabilityResults, outputWriter io.Writer) error {
+	scanResult := &scalibr.ScanResult{}
+
+	for _, source := range vulnResult.Results {
+		for _, pkg := range source.Packages {
+			scanResult.Inventories = append(scanResult.Inventories, pkg.Package.Inventory)
+		}
+	}
+
+	doc := converter.ToSPDX23(scanResult, converter.SPDXConfig{})
```

Comment:
Let's add a follow up TODO here for configuration, though that's probably some time away.
```suggestion
  // TODO(#1783): Allow user configuration
	doc := converter.ToSPDX23(scanResult, converter.SPDXConfig{})
```

Reply:
whoops yeah good call, as I'd completely forgotten about the config myself 😅